src/app-layout/visual-refresh/drawers.tsx

```diff
@@ -286,21 +255,22 @@ function DesktopTriggers() {
           );
         })}
 
-        {overflowItems && overflowItems.length > 0 && (
+        {overflowItems.length > 0 && (
           <OverflowMenu
-            drawersRefs={drawersRefs}
-            className={clsx(
-              styles['drawers-trigger'],
-              styles.trigger,
-              styles['drawers-trigger-overflow'],
-              overflowItemHasBadge && styles.badge
```

Comment:
It looks like we've lost the badge on the overflow menu when one of its items has a badge
<img width="123" alt="image" src="https://github.com/cloudscape-design/components/assets/9701338/d13fe666-6773-462a-81b8-94d19c290dff">

Reply:
Just finished annotating this PR with comments.

This change is sort of intended, pointed in another comment: https://github.com/cloudscape-design/components/pull/1444#discussion_r1293588290